crmsh/bootstrap.py

```diff
@@ -220,6 +227,8 @@ def _validate_nodes_option(self):
         crm_conf_hosts_option = self.current_user + '@' + utils.this_node()
         for user, node in zip(self.user_list, self.node_list):
             crm_conf_hosts_option += ', ' + user + '@' + node
+        config.set_option('core', 'hosts', crm_conf_hosts_option)
+        config.save()
```

Comment:
The py code under `crmsh/crmsh` directory should move out of this commit, which purpose is the behavior functional test

And for the commit log format, please starts with `Dev: behave: `

Reply:
And, for the unit test code in `test/unittests/` please use a separate commit starts with `Dev: unittest:` 
Thanks!

Reply:
Allright

Reply:
> And, for the unit test code in `test/unittests/` please use a separate commit starts with `Dev: unittest:` Thanks!

Allright